garden/public/index.html

```diff
@@ -25,6 +25,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans&display=swap" rel="stylesheet">
```

Comment:
are these for fonts? Is there a way to do this so we dont have to rely on CDNs? Could we host the files locally? Or is it too big/not best practice?

Reply:
Currently looking into this, I was able to download the font and the file size does not seem too big (total of about 650KB). Just going to have to do some playing around with the tailwind.config file to make sure it works as intended, I will let you know when that is done.